commcare-cloud/commcare_cloud.py

```diff
@@ -272,7 +272,7 @@ def git_branch():
     cwd = os.path.expanduser('~/.commcare-cloud/ansible')
     git_branch_output = subprocess.check_output("git branch", cwd=cwd, shell=True).strip().split('\n')
     starred_line, = [line for line in git_branch_output if line.startswith('*')]
-    if re.search(r'\* \(HEAD detached at .*\)', starred_line):
+    if re.search(r'\* \(.*detached .*\)', starred_line):
```

Comment:
Any particular reason to do this that came up?

Reply:
Yes, on enikshay the line was

`* (detached from origin/dm/riak-config-x-backend)`